src/phylum/ci/cli.py

```diff
@@ -121,6 +121,15 @@ def get_args(args: Optional[Sequence[str]] = None) -> tuple[argparse.Namespace,
             may not contain strict dependencies. In these cases, it is best to specify an explicit dependency file path.
             """,
     )
+    analysis_group.add_argument(
+        "-e",
+        "--exclude",
+        action="append",
+        nargs="*",
+        help="""Glob-style exclusion patterns. Ignored when dependency files are specified explicitly by argument.
+        Specify relative patterns and in quotes to prevent shell globbing. Matching is done from the right of paths. The
```

Comment:
Quotes don't prevent shell globbing though, right? Only single quotes?

Reply:
As far as Python is concerned for argument parsing, either single or double quotes will be treated the same...and both converted to single quoted lists of patterns for use internally:

```
## Using single quotes
❯ poetry run phylum-ci -vvafe 'tests/*'
...
DEBUG    Exclusion patterns provided as arguments: ['tests/*']
...

## Using double quotes
❯ poetry run phylum-ci -vvafe "tests/*"
...
DEBUG    Exclusion patterns provided as arguments: ['tests/*']
...

## Not quoting at all
❯ poetry run phylum-ci -vvafe tests/*
...
DEBUG    Exclusion patterns provided as arguments: ['tests/__init__.py', 'tests/__pycache__', 'tests/conftest.py', 'tests/constants.py', 'tests/fixtures',
         'tests/functional', 'tests/unit']
...
```

The language of `quotes` was used here instead of `double quotes` or `single quotes` because it may be necessary to use one style over the other, depending on the environment where they are specified (e.g., nesting quotes).